src/fragments/start/getting-started/reactnative/setup.mdx

```diff
@@ -20,7 +26,7 @@ import rnVersionCallout from '/src/fragments/common/react-native-version-deploym
 Install the necessary dependencies by running the following command:
 
 ```sh
-npm install aws-amplify @aws-amplify/react-native @aws-amplify/rtn-web-browser @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values
+npm install aws-amplify @aws-amplify/react-native @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values
```

Comment:
This is only needed with `signInWithRedirect`. This is the only place in the getting started that has this and we do call out the need for it on the relevant page